ja/homepage/modes_statically_generated.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 静的ファイルの生成
+content_title: 静的ファイルの生成 (プリレンダリング)
+---
+静的サイト生成は現在非常に注目されているトピックです。
+他のフレームワークに切り替える代わりにそれに慣れるのに時間を費やす、なぜ一石二鳥ではいけないのでしょうか？
```

Comment:
ここでの`Instead of`は`switching`と`spending`両方にかかっていると思うので以下のような訳はいかがでしょうか
```suggestion
別のフレームワークに切り替えて慣れるのに時間を費やす代わりに、なぜ一石二鳥ではいけないのでしょうか？
```
また、`静的サイト生成は現在非常に注目されているトピックです。`で改行せず繋げてもえると:pray: